app/javascript/mastodon/features/account_featured/components/empty_message.tsx

```diff
@@ -15,13 +18,30 @@ export const EmptyMessage: React.FC<EmptyMessageProps> = ({
   hidden,
   blockedBy,
 }) => {
+  const isMe = useAppSelector((state) => state.meta.get('me') === accountId);
```

Comment:
You can `import { me } from 'mastodon/initial_state` to skip the selector stuff.